conclave-host/src/main/kotlin/com/r3/conclave/host/internal/gramine/GramineEnclaveHandle.kt

```diff
@@ -91,11 +91,14 @@ class GramineEnclaveHandle(
          */
         val command = prepareCommand(port)
 
-        gramineProcess = ProcessBuilder()
-            .inheritIO()
+        val gramineProcessBuilder = ProcessBuilder()
             .directory(workingDirectory.toFile())
             .command(command)
-            .start()
+
+        if(enclaveMode != EnclaveMode.RELEASE)
+            gramineProcessBuilder.inheritIO()
```

Comment:
space after `if` and braces please.

But more importantly, can you add a comment why this is here.